src/dguweb/web/router.ex

```diff
@@ -45,7 +46,10 @@ defmodule DGUWeb.Router do
   end
 
   # Other scopes may use custom stacks.
-  # scope "/api", DGUWeb do
-  #   pipe_through :api
-  # end
+  scope "/api", DGUWeb do
+     pipe_through :api
+
+     get "/autocomplete", AutocompleteController, :query
+
```

Comment:
I'm not a big fan of conneg either *but* if it's your preference I don't have too many reasons why I won't do it.  The browser testing thing that @davidread is true, but that route is only there temporarily anyway.  Not too worried about caching for the prototype, and I guess we're here to try things out, so I'll update the PR and see how it feels.